packages/ui/src/common/providers/onboarding/types.ts

```diff
@@ -0,0 +1,16 @@
+interface MembershipOnBoarding extends UseMembershipOnBoarding {
+  setFreeTokens?: (value: string) => void
+  isModalOpen: boolean
+  toggleModal: () => void
+}
+
+export interface UseOnBoarding {
+  membership: MembershipOnBoarding
+}
```

Comment:
I think that designing an interface without a clear need should be at least postponed. Right now we only have one concept of onboarding, and I'm not 100% sure that we would have another in the future.

Reply:
Fixed